src/components/presentations/UserProfile/UserProfile.jsx

```diff
@@ -96,6 +108,8 @@ Userprofile.propTypes = {
   userProfile: PropTypes.shape().isRequired,
   articles: PropTypes.shape().isRequired,
   isReviewer: PropTypes.bool.isRequired,
+  getReportedArticle: PropTypes.func.isRequired,
+  reportedArticles: PropTypes.shape().isRequired,
```

Comment:
Is it possible to define what the shape of the `reportedArticles` object is? Can we be more explicit here? @sojida

Reply:
I'll update that. Thanks